### ISSUE TITLE
Feature request: Add option to sync for unidirectional synchronization

### ISSUE THREAD

**Issue by rsbrux** [meganz/MEGAcmd#638]
For the time being, since backup is still in beta, sync could be used as at least a partial workaround if it had an option for one-way synchronization (e.g. #upload only). #498

**Comment by realtebo** [comment]
Yes !!!! It's a MUST. We do not want that deleting a file from mega could delete a file on hard disk !!!!!

**Comment by angelds-mega** [comment]
Hello,

MEGAsync does not currently offer the option to one-way sync however our developers are working on this feature, which will be available in a future release. MEGAsync works as a two-way sync, meaning whatever changes (add/delete/edit) you make in the local Sync folder gets reflected in your Cloud Drive Sync folder.

Currently, as a workaround, you can achieve one-way syncing by using the manual "Upload" and "Download" features.

You can use the "Upload" feature from the MEGAsync user interface to manually upload the same file/folder to the same destination folder in your MEGA Cloud Drive.

MEGAsync will detect the files that are already there and only upload the new or modified ones. This way you can delete the folder on your computer afterward and this action won't be reflected in your Cloud Drive.

In the same way, the "Download" feature will allow you to download files/folders without syncing them. This way you can delete the folder in your Cloud Drive afterward and this action won't be reflected in your locally synced folder.

As an alternative, you can simply delete the sync path after syncing.
Sync your folder and, once the synchronisation has completed, go to Settings/Preferences - Sync and manually un-tick the nominated sync path or delete it.

You can then delete the folder/file on your computer and this action won't be reflected on your Cloud Drive, or vice-versa.

**Comment by rsbrux** [comment]
> Currently, as a workaround, you can achieve one-way syncing by using the manual "Upload" and "Download" features.
> 
> You can use the "Upload" feature from the MEGAsync user interface to manually upload the same file/folder to the same destination folder in your MEGA Cloud Drive.
> 
> MEGAsync will detect the files that are already there and only upload the new or modified ones. This way you can delete the folder on your computer afterward and this action won't be reflected in your Cloud Drive.
> 
> In the same way, the "Download" feature will allow you to download files/folders without syncing them. This way you can delete the folder in your Cloud Drive afterward and this action won't be reflected in your locally synced folder.
> 

I need to do this unattended.  Can  the megacmd commands [mega_put](https://github.com/meganz/MEGAcmd/blob/master/UserGuide.md#put) and [mega-get](https://github.com/meganz/MEGAcmd/blob/master/UserGuide.md#get) be used instead of the manual upload and download operations you describe?

> As an alternative, you can simply delete the sync path after syncing. Sync your folder and, once the synchronisation has completed, go to Settings/Preferences - Sync and manually un-tick the nominated sync path or delete it.
> 
> You can then delete the folder/file on your computer and this action won't be reflected on your Cloud Drive, or vice-versa.
Is there some equivalent operation to the describe operation in Settings/Preferences using megacmd?

**Comment by disbit** [comment]
Would love this feature! Am also on a premium plan fwiw.

**Comment by barto95100** [comment]
Any news for this feature ?
